code/ZAS/_gas_mixture.dm

```diff
@@ -45,6 +45,8 @@
 	if(!moles)
 		return
 
+	if(!src[gasid])
+		src[gasid] = 0
```

Comment:
You don't have to do this. First, because `operator[]` already does it, and second, because it works fine if it's null anyway.

Reply:
Oh, right. I was trying to figure out why adding GAS_N2O wasn't working.